src/backend/marsha/core/api/live_registration.py

```diff
@@ -118,6 +129,29 @@ def get_throttles(self):
 
         return [throttle() for throttle in throttle_class]
 
+    def perform_create(self, serializer):
+        """Overrides perform_create to send the mail and catch error if needed"""
+        super().perform_create(serializer)
+        try:
+            msg_plain = render_to_string("core/mail/text/sample.txt")
+            msg_html = render_to_string("core/mail/html/sample.html")
+            send_mail(
+                _("mail_registration_object"),
+                msg_plain,
+                settings.EMAIL_FROM,
+                [serializer.validated_data["email"]],
+                html_message=msg_html,
+                fail_silently=False,
+            )
+        except smtplib.SMTPException as exception:
+            # no exception raised as user can't sometimes change his mail,
+            logger.warning(
+                "registration mail %s not send", serializer.validated_data["email"]
+            )
+            capture_message(
```

Comment:
capturing only the exception is most useful I think. We will have the stacktrace with it

Reply:
And we will also have the exact exception type